src/slic3r/GUI/PartPlate.cpp

```diff
@@ -470,7 +470,7 @@ void PartPlate::calc_gridlines(const ExPolygon& poly, const BoundingBox& pp_bbox
 	int count = 0;
 	int step  = 10;
 	// Orca: use 500 x 500 bed size as baseline.
-    auto      grid_counts = pp_bbox.size() / ((coord_t) scale_(step * 50));
+    const Point grid_counts = pp_bbox.size() / ((coord_t) scale_(step * 50));
```

Comment:
This line caused false address sanitizer exceptions that prevented me from debugging the real poblem so I fixed this. Doesn't really related to the crash though.